### DISCUSSION TITLE
Display pointcloud (.pcd, .ply) issue (huge point size) in CVAT

### DISCUSSION THREAD

**Discussion by graf11** [cvat-ai/cvat#4248] [Q&A]
I got .pcd (tried .ply also) file from D435 Intel RealSense camera looks like this: 
<a href="https://www.upload.ee/image/13830009/4.png"><img src="https://www.upload.ee/thumb/13830009/4.png" border="0" alt="4.png" /></a>
but when im uploading it in cvat it looks like this(huge point size):(click for full size screenshot)
<a href="https://www.upload.ee/image/13830021/Screenshot_2022-01-21_143144__3_.png"><img src="https://www.upload.ee/thumb/13830021/Screenshot_2022-01-21_143144__3_.png" border="0" alt="Screenshot_2022-01-21_143144__3_.png" /></a>
ive tried my pointclouds on other soft - its dysplayed fine.
how am i able to decrease point size in CVAT? or what else should i do? any advice guys? it is unreal to label this

**Comment by nmanovic** [comment]
@graf11 , thanks for the report. Could you please submit an issue? If you can please attach your pcd file.

**Comment by graf11** [comment]
sure, ive tried plenty of pcd's and ply's also, on the cvat.org and in docker installed on ubuntu version, this is one from the screenshot: https://www.upload.ee/files/13833163/4.pcd.html

**Comment by graf11** [comment]
hello, **nmanovic**, any luck in this issue? if you need more information about problem,  ill provide it

**Comment by nmanovic** [comment]
@graf11 , we don't have resources to look into the problem right now. Could you please create an issue and attach all relevant information? We will add the issue into our roadmap and address in the future releases. Sorry for the bad experience. Unfortunately we cannot process all requests. If you have some developer experience, we are welcome to accept contribution with the fix. CVAT is an open source product. Thus you always can change something or fix critical issues for you.
